renku/cli/_format/datasets.py

```diff
@@ -37,6 +37,8 @@ def tabular(client, datasets):
                 ('created', None),
                 ('creators_csv', 'creators'),
             )),
+            # workaround for tabulate issue 181
```

Comment:
What do you mean by issue #181?

Reply:
This issue: https://bitbucket.org/astanin/python-tabulate/issues/181/disable_numparse-fails-on-empty-input

I found that yesterday. I originally saw that version was `1` instead of `1.0` for dataverse, and thought tabluate formatting numbers was responsible for it, so I added this option, but the issue was somewhere else.

I left the option in as it makes sense to not treat certain things as number, as this can lead to weird behaviour (e.g. hashes ending in e1 being treated as numbers, which is a case mentioned in the tabulate docs for this feature)

Reply:
AHA! Ok, cool tnx! Put the link in the comment as well, so we can keep track of it and make a ticket here on our tracker.